src/utilities/hasSCOBeenUpdated.js

```diff
@@ -0,0 +1,11 @@
+import isEqual from 'lodash/isEqual';
```

Comment:
You don't need lodash/isEqual. You can either use

`return JSON.stringify(defaultSCOState) === JSON.stringify(SCOcomparator);`
OR
```
export const convertObjectToBase64 = (obj) => Buffer.from(JSON.stringify(obj)).toString('base64');

return convertObjectToBase64(defaultSCOState) === convertObjectToBase64(SCOcomparator);
```

Where convertObjectToBase64 is already in a utility function. This technically does not matter for this though since you are not making a bunch of copies of the JSON string. If this were getting passed all over the place, then we may care about the memory...

Reply:
Oh yeah, the reasoning here -- lodash/isEqual is 1600 lines of code and that is a lot. isEqual handles things like arrays out of order and whatnot. You do not care because it is either the same JSON as the default, or it is different. So this will be smaller, and maybe run faster.

Reply:
Done!